pkg/apis/api.acorn.io/v1/types.go

```diff
@@ -342,6 +342,7 @@ type Config struct {
 	SetPodSecurityEnforceProfile *bool          `json:"setPodSecurityEnforceProfile" usage:"Set the PodSecurity profile on created namespaces (default true)"`
 	PodSecurityEnforceProfile    string         `json:"podSecurityEnforceProfile" usage:"The name of the PodSecurity profile to set (default baseline)" wrangler:"nullable"`
 	DefaultPublishMode           v1.PublishMode `json:"defaultPublishMode" usage:"If no publish mode is set default to this value (default user)" wrangler:"nullable,options=all|none|defined"`
+	HttpEndpointPattern          *string        `json:"httpEndpointPattern" wrangler:"nullable" name:"http-endpoint-pattern" usage:"Go template for how to format that endpoint's http domain. Valid variables to use are: App, Container, Namespace, Hash and ClusterDomain. (default pattern is {{.Container}}-{{.App}}-{{.Hash}}.{{.ClusterDomain}})"`
```

Comment:
- can you put wrangler at the end, like the others?
- "..for how to fomrat that endpoint's" sounds weird 
```suggestion
	HttpEndpointPattern          *string        `json:"httpEndpointPattern" name:"http-endpoint-pattern" usage:"Go template for how to format apps' http endpoints. Valid variables to use are: App, Container, Namespace, Hash and ClusterDomain. (default pattern is {{.Container}}-{{.App}}-{{.Hash}}.{{.ClusterDomain}})" wrangler:"nullable"`
```

Reply:
just did that in browser, so not tested

Reply:
oh and if you take it, youll have to re-gen docs too